nixos/modules/system/boot/networkd.nix

```diff
@@ -28,6 +29,7 @@ let
         (assertValueOneOf "IPv6PrivacyExtensions" (boolValues ++ ["prefer-public" "kernel"]))
         (assertValueOneOf "IPv4Forwarding" boolValues)
         (assertValueOneOf "IPv6Forwarding" boolValues)
+        (assertValueOneOf "UseDomains" boolValues ++ ["route"])
```

Comment:
This way without the parentheses around `boolValues ++ ["route"]` breaks, as it tries to combine the function with the list. Can you add the parentheses like in line 29, please?

For the future: You can also run a nixos test for systemd networkd with `nix-build -A nixosTests.systemd-networkd` in the nixpkgs directory

Reply:
Thanks, missed that on my first fix. Should be fixed now & the test you suggested ran successfully :+1: